lib/src/git.rs

```diff
@@ -1273,7 +1273,10 @@ pub fn fetch(
         .map(|pattern| {
             pattern
                 .to_glob()
-                .filter(|glob| !glob.contains(INVALID_REFSPEC_CHARS))
+                .filter(
+                    /* This check will fail if `to_glob()` escapes a literal `*` */
```

Comment:
nit: "check will fail" sounds like the check doesn't work. Remove the comment? or say something like "to_glob() should escape * as [*] if the pattern wasn't a glob"?

Reply:
I edited it a bit, hope it's better.